### ISSUE TITLE
Mission 8: FRUSTRATIONS when hitting Cerberi base (this needs changing)

### ISSUE THREAD

**Issue by Feared1** [BlackDragonN001/FERemastered#75]
There is a Hadean Arsenal called "Damaged Weapons Vat" in the Cerberi base that has messed up collisions and all my units tried to attack it but it has infinite health

**Comment by Feared1** [comment]
This mission is incredibly long and unforgiving; it is almost impossible to complete without trial/error savegames (especially to find the excluders at the end)

Exlcuders should be moved to the front area ONLY, unprotected by infinitely-building OP gun towers; this was ridiculous and frustrating... LITERALLY impossible without saves (and I am VERY good at this game)... if you want to keep it the same, add an objective while you are in the Eyes of Xyr to NAV the Excluders in the Cerberi base so you know where they are, but it would still be immensely challenging to destroy them all.

**Comment by JJ173** [comment]
I am not sure changing the mission length or adding objectives would solve the problem. I would say we should make sure that the damaged Hadean buildings should be changed to team 0 here.

**Comment by Feared1** [comment]
The Excluders should not be buried in the back of the base as they currently are. This was the biggest frustration as they are impossible to find without knowing where they are beforehand, and you cannot send your units in effectively if there is even one Excluder up. Maybe this is the way it was in 1.2, but this is a very simple improvement and makes the mission actually possible without prior knowledge of how to do it (which should always be the case, EVERY time).

**Comment by jack775544** [comment]
I agree, I remember doing this mission not to long ago and there was a lot of sitting back using the extended range of the prototype tank to take down gun towers while hunting for the jammers, which took forever and rather boring.

**Comment by Malifious** [comment]
Older thread but I made a note about this in my bug reports that I submitted a few days ago.
The excluder jammers are a part of the Cerberi build list and it makes sense for them to have some in their base but as Jack pointed out, you're essentially sitting back and sniping things with the prototype tank while a service truck heals you. It's boring, slow, and defeats the purpose of building some big army to rescue Schultz. 

If anything, place a single excluder at the front (there's one by the Cerberi Spikes) and another perhaps to the right of the Xyr Tower when you first enter the base. Having excluders at the back of the base is a bad move.
